README.md

```diff
@@ -1,23 +1,39 @@
 # XenoCP
 
+XenoCP is a tool for cleansing mouse reads in xenograft BAMs.
+XenoCP can be easily incorporated into any workflow, as it takes a BAM file
+as input and efficiently cleans up the mouse contamination. The output is a clean
+human BAM file that could be used for downstream genomic analysis. 
+
 ## Getting started
 
-	git clone https://github.com/stjude/XenoCP.git
-	cd XenoCP
-	gradle :xenocp:installDist
+XenoCP can be run in the cloud on DNAnexus at
+https://platform.dnanexus.com/app/stjude_xenocp
```

Comment:
Is there a risk people will have permissions issues with this?  I seem to remember something about the app being available to people with a St. Jude Cloud users group of some sort that they got automatically after logging into St. Jude Cloud Platform, or something along those lines.

Reply:
@jsunny23 - Can you comment on this?

The dxapp config has the following set of users. 
`  "authorizedUsers": [
      "user-lding",
      "user-appdpdnanexus_stjude.org",
      "user-appdddnanexus_stjude.org",
      "org-stjude_cloud_app_runners",
      "org-stjude_cloud"
  ],`

Reply:
Yes, when a user logs into St. Jude Cloud Platform, they're added to `org-stjude_cloud_app_runners`. So all St. Jude Cloud users should be able to see it under the tools page (I can see the app) and be vendable from platform. 

Note: DNAnexus only allow developers or authorized users to run apps. So if you want this to be truly public, that may require talking to DNAnexus.

Reply:
I think you should probably specify that you'll need to login to St. Jude Cloud at least once to get access to the app.

Reply:
For reference, when you try to make an app `PUBLIC` on DNAnexus:

```
***
WARNING: app-mtsg will be made public. Anyone will be able to view and run all
published versions of this app.
***
Confirm making this app public [y/N]: y
Error: You are not authorized to add "PUBLIC" to the authorized users list for app mtsg; if you wish to make an app or a workflow available to all users, then please contact support, code 401.
```

Reply:
I have contacted DNAnexus support.  Their response led me to believe this is going to be a slow process.  In the meantime, I have added instructions on how to login to St. Jude Cloud to get access to the app at the permalink page for the paper:

https://pecan.stjude.cloud/permalink/xenocp

Could somebody from the cloud team make sure the instructions I wrote are correct?  If so, we can update the README to point to that permalink page.  Then, once it's truly public, we can go back and update again.